moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1506,7 +1506,7 @@ as the new values that correspond to the group */
   }
 
   bool satisfiesBounds(double margin = 0.0) const;
-  bool satisfiesBounds(const JointModelGroup* joint_group, double margin = 0.0) const;
+  bool satisfiesBounds(const JointModelGroup* group, double margin = 0.0) const;
```

Comment:
If you change the names here anyway, wouldn't it make sense to streamline this file to `JointModelGroup* jmg`?

Reply:
This PR is mainly about automatic changes. I wouldn't mix in such manual changes. Actually I suggest to drop these variable renamings, because there not always satisfactory.

Reply:
i think we shouldn't be too picky about the power of clang-tidy and how much time it saves us from doing these optimizations and cleanup tasks manually...

Reply:
I agree with rhaschke, I'll drop the `readability-inconsistent-declaration-parameter-name` check for now, and make another PR that will manually resolve and use the 'better' name if it comes from the header (clang-tidy defaults to using the name used in the cpp file).  

Once that is finished, we can add `readability-inconsistent-declaration-parameter-name` back to the checks to ensure no new changes are added that use different names.